cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -23,6 +36,61 @@ public Intellij(IdeContext context) {
     super(context, "intellij", Set.of(Tag.INTELLIJ));
   }
 
+  @Override
+  protected void runIde(String... args) {
```

Comment:
dont you want to override `runTool` here if that method by default does not behave correct for Intellij?